examples/custom-tooltips-demo/index.js

```diff
@@ -0,0 +1,89 @@
+/**
+ * @license
+ * Copyright 2021 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @fileoverview Test page for example plugin showing custom tooltip rendering.
+ */
+
+
+/**
+ * Create and register the custom tooltip rendering function.
+ * This could be extracted into a plugin if desired.
+ */
+function initTooltips() {
+  // Create a custom rendering function. This function will be called whenever
+  // a tooltip is shown in Blockly. The first argument is the div to render
+  // the content into. The second argument is the element to show the tooltip
+  // for.
+  const customTooltip = function(div, element) {
+    if (element instanceof Blockly.BlockSvg) {
+      // You can access the block being moused over.
+      // Here we get the color of the block to set the background color.
+      div.style.backgroundColor = element.getColour();
+    }
+    const tip = Blockly.Tooltip.getTooltipOfObject(element);
+    const text = document.createElement('div');
+    text.textContent = tip;
+    const container = document.createElement('div');
+    container.style.display = 'flex';
+    // Check to see if the custom property we added is present.
```

Comment:
Maybe add information on where it was added? '...we added in the block definition is present.'